crates/cheatnet/src/rpc.rs

```diff
@@ -661,6 +664,70 @@ pub fn library_call_syscall(
     })
 }
 
+pub enum KeccakOutput {
+    Success { ret_data: KeccakResponse },
+    Panic { panic_data: Vec<Felt252> },
+}
+
+// Inspired by blockifier::execution::syscalls::mod::keccak
+pub fn keccak_syscall(
```

Comment:
Why we cannot redirect execution to `original_cairo_hint_processor`? It has keccak implemented and I would prefer us to avoid maintaining hash functions implementations even it is only for test purposes.